pkgs/by-name/au/authentik/package.nix

```diff
@@ -0,0 +1,248 @@
+          dacite
+          daphne
+          deepmerge
+          defusedxml
+          django
+          django-filter
+          django-guardian
+          django-model-utils
+          django-prometheus
+          django-redis
+          djangorestframework
+          djangorestframework-guardian2
+          docker
+          drf-spectacular
+          duo-client
+          facebook-sdk
+          flower
+          geoip2
+          gunicorn
+          httptools
+          kubernetes
+          ldap3
+          lxml
+          opencontainers
+          packaging
+          paramiko
+          psycopg
+          pycryptodome
+          pydantic
+          pydantic-scim
+          pyjwt
+          pyyaml
+          requests-oauthlib
+          sentry-sdk
+          structlog
+          swagger-spec-validator
+          twilio
+          twisted
+          ua-parser
+          urllib3
+          uvicorn
+          uvloop
+          watchdog
+          webauthn
+          websockets
+          wsproto
+          xmlsec
+          zxcvbn
+          jsonpatch
+        ] ++ [
+          codespell
+        ];
+
+        postInstall = ''
+          mkdir -p $out/web $out/website
+          cp -r lifecycle manage.py $out/${final.python.sitePackages}/
+          cp -r blueprints $out/
+          cp -r ${webui}/dist ${webui}/authentik $out/web/
+          cp -r ${website} $out/website/help
+          ln -s $out/${final.python.sitePackages}/lifecycle $out/lifecycle
+        '';
+      };
+    };
+  };
+
+  authentik-django = python.pkgs.authentik-django;
+
+  proxy = buildGoModule {
+    pname = "authentik-proxy";
+    inherit version;
+
+    src = runCommand "authentik-proxy-source" {} ''
+      mkdir -p $out/web $out/authentik
+
+      cp -r ${src}/cmd ${src}/internal $out/
+      cp -r ${src}/authentik/lib $out/authentik/
+      cp ${src}/web/static.go $out/web/
+
+      cp ${src}/web/robots.txt ${src}/web/security.txt $out/web/
+      cp ${src}/go.mod ${src}/go.sum $out/
+    '';
+
+    postPatch = ''
+      substituteInPlace internal/gounicorn/gounicorn.go \
+        --replace './lifecycle' "${authentik-django}/lifecycle"
+      substituteInPlace web/static.go \
+        --replace './web' "${authentik-django}/web"
+      substituteInPlace internal/web/static.go \
+        --replace './web' "${authentik-django}/web"
+    '';
+
+    CGO_ENABLED = 0;
+
+    vendorHash = "sha256-8F9emmQmbe7R+xtGrjV5ht0adGasU6WAvLa8Wxr+j8M=";
+
+    postInstall = ''
+      mv $out/bin/server $out/bin/authentik
+    '';
+
+    subPackages = [ "cmd/server" ];
+  };
+
+in stdenvNoCC.mkDerivation {
+  pname = "authentik";
+  inherit src version;
+
+  postPatch = ''
+    rm Makefile
+    patchShebangs lifecycle/ak
+
+    # This causes issues in systemd services
+    substituteInPlace lifecycle/ak \
+      --replace 'printf' '>&2 printf' \
+      --replace '> /dev/stderr' ""
+  '';
+
+  installPhase = ''
+    mkdir -p $out/bin
+    cp -r lifecycle/* $out/bin/
```

Comment:
It seems the following files are not wrapped nor are their shebang patched.

* results/authentik/bin/migrate.py
* results/authentik/bin/wait_for_db.py

Should they even be in `bin`? Their names are pretty generic.